src/fidesctl/ctl/connectors/aws.py

```diff
@@ -82,6 +82,20 @@ def describe_rds_instances(client: Any) -> Dict[str, List[Dict]]:  # type: ignor
     return describe_instances
 
 
+@handle_common_aws_errors
+def get_tagging_resources(client: Any) -> List[str]:  # type: ignore
```

Comment:
is there any way to put an actual type here?

Reply:
The boto3 library does not return a typed client when calling `boto3.client` unfortunately. 

https://github.com/boto/botocore/issues/2297